ldoc.lua

```diff
@@ -171,7 +171,6 @@ doc.ldoc = ldoc
 
 -- if the corresponding argument was the default, then any ldoc field overrides
 local function override (field,defval)
-   defval = defval or false
```

Comment:
This line was to cast possible `nil` values to a boolean `false`. I think it was probably correct.

Reply:
See also #118.